auth/auth.go

```diff
@@ -5,13 +5,32 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/url"
-	"strings"
 
 	"github.com/G-Node/gin-cli/util"
 	"github.com/G-Node/gin-cli/web"
 	"github.com/G-Node/gin-core/gin"
+	"net/http"
```

Comment:
Well as you check for http response codes (like ==200) in the code it could be argued that it kind of belongs here.